### PR TITLE
Pipeline updates

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: .github/workflows/depoy-prod-start.yml [hunk 1]
```diff
@@ -70,3 +74,74 @@ jobs:
...
+        uses: redhat-actions/oc-login@v1
+        with:
+          openshift_server_url: ${{ env.OPENSHIFT_SERVER }}
+          openshift_token: ${{ env.OPENSHIFT_TOKEN }}
+          insecure_skip_tls_verify: true
+          namespace: 3cd915-prod
+      - name: call scripts to sync mayan
+        shell: bash
+        run: |
+          oc process -f ./openshift/4.0/templates/jobs/mayan-sync.yaml -p NAMESPACE=3cd915-prod -p TOKEN_URL=https://loginproxy.gov.bc.ca:443/auth/realms/standard/protocol/openid-connect/token -p CLIENT_ID=property-services-project-api-4380 -p MAYAN_SYNC_URL=https://https://pims.th.gov.bc.ca/:443/api/documents/sync/mayan/metadatatype -p KEYCLOAK_SECRET_NAME=pims-api-sso | oc create -f - | grep -oP "(?<=\/)[^\s]*" | (read TASK_NAME; oc wait --for=condition=succeeded taskruns/$TASK_NAME --timeout=80s)
```
Comment (FuriousLlama) [inline .github/workflows/depoy-prod-start.yml:120]: I noticed that the mayan url is hardcoded. Should it use a config for that too?

**Review by FuriousLlama — APPROVED** [review]

File: .github/workflows/depoy-prod-start.yml (hunk 1 above)
Comment (devinleighsmith) [reply .github/workflows/depoy-prod-start.yml:120]: In the interest of time, I'll log a follow up task for that, as well as allowing the base mayan sync and keyclaok sync to be parameterized.

Comment (asanchezr) [inline .github/workflows/depoy-prod-start.yml:120]: hmm - is this URL correct?

`MAYAN_SYNC_URL=https://https://pims.th.gov.bc.ca/:443/api/documents/sync/`

not sure `/:443` is right

Comment (asanchezr) [reply .github/workflows/depoy-prod-start.yml:120]: You prob want `https://pims.th.gov.bc.ca:443/api/....`

File: .github/workflows/retag-test-to-uat.yml [hunk 2]
```diff
@@ -86,10 +87,59 @@ jobs:
...
+        uses: redhat-actions/oc-login@v1
+        with:
+          openshift_server_url: ${{ env.OPENSHIFT_SERVER }}
+          openshift_token: ${{ env.OPENSHIFT_TOKEN }}
+          insecure_skip_tls_verify: true
+          namespace: 3cd915-test
+      - name: call scripts to sync mayan
+        shell: bash
+        run: |
+          oc process -f ./openshift/4.0/templates/jobs/mayan-sync.yaml -p NAMESPACE=3cd915-test -p TOKEN_URL=https://test.loginproxy.gov.bc.ca:443/auth/realms/standard/protocol/openid-connect/token -p CLIENT_ID=property-services-project-api-4380 -p MAYAN_SYNC_URL=https://https://uat-pims.th.gov.bc.ca/:443/api/documents/sync/mayan/metadatatype -p KEYCLOAK_SECRET_NAME=pims-api-sso-uat | oc create -f - | grep -oP "(?<=\/)[^\s]*" | (read TASK_NAME; oc wait --for=condition=succeeded taskruns/$TASK_NAME --timeout=80s)
```
Comment (asanchezr) [inline .github/workflows/retag-test-to-uat.yml:109]: mayan sync urls are malformed here as well

**Pull request merged**